src/fiskaltrust.Launcher.Common/Configuration/Configuration.cs

```diff
@@ -141,11 +156,63 @@ public LauncherConfiguration Redacted()
 
             return redacted;
         }
+
+        public static LauncherConfiguration Deserialize(string text)
+        {
+            var configuration = JsonSerializer.Deserialize(text, typeof(LauncherConfiguration), SerializerContext.Default) as LauncherConfiguration ?? throw new Exception($"Could not deserialize {nameof(LauncherConfiguration)}");
+            configuration.SetAlternateNames(text);
+            return configuration;
+        }
+
+        public string Serialize() => JsonSerializer.Serialize(this, typeof(LauncherConfiguration), SerializerContext.Default);
+
+        internal void SetAlternateNames(string text)
```

Comment:
Yes ^^ basically I wanted to have two valid property names for the cashboxid so I created this `AlternateNameAttribute` and added it here:

```cs
        [JsonPropertyName("cashboxId")]
        [AlternateName("ftCashBoxId")]
        public Guid? CashboxId { get => _cashboxId; set => _cashboxId = value; }
```

The `SetAlternateNames` method is used to get these `AlternateNames` from the json and merge them into the object if not set already.

A bit overengineered/overkill but the best solution I found ^^

> I want to have both names because it is called `ftCashBoxId` in the cashbox config but all other launcher config keys have a camelCase version of the kebab-case cli parameter which would be `cashboxId`